frontend/src/app/components/filters/query-filter/query-filter.component.html

```diff
@@ -75,6 +73,13 @@
                            [shouldFocus]="shouldFocus"
                            [filter]="filter"></filter-string-value>
 
+      <filter-searchable-multiselect-value *ngSwitchCase="'[]WorkPackage'"
+                           (filterChanged)="onFilterUpdated($event)"
+                           [shouldFocus]="shouldFocus"
```

Comment:
The FilterSearchableMultiselectValueComponent doesn't shouldFocus @input. Do we need to pass the shouldFocus from the parent QueryFiltersComponent?